### PR TITLE
implement typescript

### PR DESCRIPTION
**Pull request by anitavictoria**

### CODE REVIEW

File: src/components/Form/FormButton/ButtonContainer.tsx [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
+}
+function ButtonContainer({ safeSetState, state }: IProps) {
   const shouldShowBackButton = () => state.step > 1 && state.step < 3;
 
   const shouldShowNextButton = () => state.step < 2;
 
-  const changeStep = (stepChange) => {
+  const changeStep = (stepChange: number) => {
     safeSetState({ step: state.step + stepChange });
+    safeSetState({ email: "state.step + stepChange" });
```
Comment (janosiecki) [inline src/components/Form/FormButton/ButtonContainer.tsx:15]: delete it

File: src/components/Form/FormButton/FormButton.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { Button } from "@mui/material";
+interface IProps {
+  label: string;
+  changeStep: () => void;
```
Comment (janosiecki) [inline src/components/Form/FormButton/FormButton.tsx:4]: changeStep takes argument

File: src/components/Form/FormContainer.tsx [hunk 3]
```diff
@@ -43,21 +45,26 @@ function FormContainer() {
...
 
   return (
     <>
       <Grid container alignItems="flex-start">
-        <Formik initialValues={state}>
-          {({ values }) => (
+        <Formik
+          initialValues={state}
+          onSubmit={() => {
+            console.log("submit");
```
Comment (janosiecki) [inline src/components/Form/FormContainer.tsx:57]: Add comment that this is temporary submit method

File: src/components/Form/types.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
+export interface CurrentFormProps {
```
Comment (janosiecki) [inline src/components/Form/types.ts:1]: if you have short interface for props, it is better to keep it in component file

File: src/components/Form/types.ts [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  taxPayer: string;
+  foundingYear: string;
+  supplierCategory: string;
+  employeesAmount: string;
+  sumOfSales: string;
+  departments: string;
+  service: string;
+  equity: string;
+}
+export interface IFieldProps {
```
Comment (janosiecki) [inline src/components/Form/types.ts:30]: up

File: tsconfig.json [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "noFallthroughCasesInSwitch": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
```
Comment (janosiecki) [inline tsconfig.json:15]: add "baseUrl": "./src"  

Then you will be able to use relative path from src dir in your imports

**Review by janosieckidev — APPROVED** [review]

**Review by janosiecki — APPROVED** [review]

**Pull request merged**